We pass `--no-history` to gclient, so there will be no version for electron.
--- a/electron/script/lib/get-version.js
+++ b/electron/script/lib/get-version.js
@@ -11,7 +11,7 @@
   // The only difference in the "git describe" technique is that technically a commit can "change" it's version
   // number if a tag is created / removed retroactively.  i.e. the first time a commit is pushed it will be 1.2.3
   // and after the tag is made rebuilding the same commit will result in it being 1.2.4
-  const output = spawnSync('git', ['describe', '--tags', '--abbrev=0'], {
+  const output = spawnSync('cat', ['ELECTRON_VERSION'], {
     cwd: path.resolve(__dirname, '..', '..')
   });
   if (output.status !== 0) {
